pkg/webhook/resources/setting/validator.go

```diff
@@ -775,7 +775,7 @@ func validateSupportBundleNodeCollectionTimeoutHelper(value string) error {
 		return nil
 	}
 
-	i, err := strconv.Atoi(value)
+	i, err := webhookUtil.StrictAtoi(value)
```

Comment:
We already have negative number validation in `webhookUtil.StrictAtoi` which make L782~L784 redundant

Reply:
@brandboat I should remove negative number validation in `webhookUtil.StrictAtoi`. It's just a stronger version of `strconv.Atoi`.